sympy/series/limits.py

```diff
@@ -42,7 +42,15 @@ def limit(e, z, z0, dir="+"):
     Gruntz algorithm (see the gruntz() function).
     """
 
-    return Limit(e, z, z0, dir).doit(deep=False)
+    if dir == "+-":
+        llim = Limit(e, z, z0,dir="-").doit(deep=False)
+        rlim = Limit(e, z, z0,dir="+").doit(deep=False)
+        if llim == rlim:
+            return rlim
+        else:
+            return None
```

Comment:
I don't think None is an appropriate return type. It should either raise an exception or return a SymPy type.